tools/internal_ci/linux/grpc_e2e_performance_v2.cfg

```diff
@@ -16,7 +16,7 @@
 
 # Location of the continuous shell script in repository.
 build_file: "grpc/tools/internal_ci/linux/grpc_e2e_performance_v2.sh"
-timeout_mins: 120
+timeout_mins: 720
```

Comment:
since this is and "experimental job" the max timeout is not that important, but 720m seems excessive. I really wouldn't want to wait for an entire working day to get my results. Let's set this back to e.g. 240min max?